docs/input.rst

```diff
@@ -81,6 +81,38 @@ It is possible to access an individual sample too:
 
 Note that the equivalent method :meth:`Input.get_sample()` is also available.
 
+It is also possible to supply a member_name to :meth:`SampleIterator.get_dictionary()`,
```

Comment:
This is too much detail for this section, I think. Move the detailed explanation to data.rst and leave here only the first sentence.

You should also change the example, because data.rst uses a different type for all examples.